config/README.md

```diff
@@ -1,135 +1,139 @@
-*   [Optional] Golang. Recommended to understand go, and the go toolchain.
+    * Required if deploying to Google Cloud.
+*   Golang.
+    *   Required if developing the DSS codebase.
 
 
 ## Building Docker images
 
-The grpc-backend and http-gateway binaries are built as docker images and pushed
-to a docker registry.  Which docker registry you use will depend on your
-individual requirements and choice of cloud platform.
 
-### Google Container Registry
+The grpc-backend and http-gateway are the 2 main binaries for processing DSS 
+requests. These are both built and pushed to a docker registry of your choice.
+You can easily find out how to push to a docker registry through a quick search.
+All major cloud providers have a docker registry service, or you can set up your
+own.
 
-List existing images:
+TODO(): The linux foundation should provide a default docker registry, with
+golden build versions.
 
-    gcloud --project $CLOUD_PROJECT container images list
-
-List the tags on an existing image:
-
-    gcloud --project $CLOUD_PROJECT container images list-tags gcr.io/$CLOUD_PROJECT/http-gateway
-
-Build a new image:
-
-    docker build -f cmds/http-gateway/Dockerfile  . -t gcr.io/$CLOUD_PROJECT/http-gateway:$VERSION
-    docker build -f cmds/grpc-backend/Dockerfile  . -t gcr.io/$CLOUD_PROJECT/grpc-backend:$VERSION
-
-Push your new image to Google Container Registry:
-
-    docker push gcr.io/$CLOUD_PROJECT/http-gateway:$VERSION
-    docker push gcr.io/$CLOUD_PROJECT/grpc-backend:$VERSION
+Set the environment variable `DOCKER_URL` to your docker registry url endpoint.
 
 Use the `build.sh` script in this directory to build and push an image tagged
 with the current date and git commit hash.
 
 
 ## Creating a new Kubernetes cluster
 
-You need to create a Kubernetes cluster to run the cockroachdb instance and the
-gRPC and HTTP servers.  You can do this on any supported
-[cloud provider](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/)
-or even on your own infrastructure.  Instructions for GCP are given below,
-consult the Kubernetes documentation for other providers.
+This secion discusses deploying a kubernetes service, although you can deploy 
+a DSS node however you like as long as it meets the above requirements. You can
+do this on any supported [cloud provider](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/)
+or even on your own infrastructure.  Consult the Kubernetes documentation for
+which ever provider you choose.
 
-### Google Container Engine
 
-Create a new cluster in the given zone:
+### Deploying with Kubernetes
 
-    gcloud --project $CLOUD_PROJECT container clusters create $CLUSTER_NAME --zone $ZONE
-
-Fetch credentials for the cluster.  This populates your \~/.kube/config file
-and makes all future kubecfg commands target this cluster.
-
-    gcloud --project $CLOUD_PROJECT container clusters get-credentials $CLUSTER_NAME --zone $ZONE
```

Comment:
Let's add a step to create the Kubernetes cluster and note that a separate cluster should be used for each DSS instance.  This content can be re-added as a GCP-specific sub-bullet under that note.

Reply:
done